### PR TITLE
[shaderwriter] Update to version 1.1.0

### PR DESCRIPTION
**Pull request by DragonJoker**
**Describe the pull request**

- #### What does your PR fix?  
  Updates shaderwriter to version 1.1.0

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  No change related to that, No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Hopefully yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/shaderwriter/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "shaderwriter",
-  "version-string": "1.0.0",
+  "version-string": "1.1.0",
```
Comment (JonLiu1993) [inline ports/shaderwriter/vcpkg.json:3]: @DragonJoker, please change  "version-string" to "version".

File: versions/s-/shaderwriter.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "554f2441c891c950d17ad4b7ba036f6e20993b07",
+      "version-string": "1.1.0",
```
Comment (JonLiu1993) [inline versions/s-/shaderwriter.json:5]: please change "version-string" to "version".

File: ports/shaderwriter/vcpkg.json (hunk 1 above)
Comment (DragonJoker) [reply ports/shaderwriter/vcpkg.json]: Done

File: versions/s-/shaderwriter.json (hunk 2 above)
Comment (DragonJoker) [reply versions/s-/shaderwriter.json]: Done

**Review by JonLiu1993 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**